CMakeLists.txt

```diff
@@ -200,428 +200,8 @@ set(ZIG_CPP_SOURCES
-    "${CMAKE_SOURCE_DIR}/src/codegen/llvm.zig"
-    "${CMAKE_SOURCE_DIR}/src/codegen/llvm/bindings.zig"
-    "${CMAKE_SOURCE_DIR}/src/glibc.zig"
-    "${CMAKE_SOURCE_DIR}/src/introspect.zig"
-    "${CMAKE_SOURCE_DIR}/src/libc_installation.zig"
-    "${CMAKE_SOURCE_DIR}/src/libcxx.zig"
-    "${CMAKE_SOURCE_DIR}/src/libtsan.zig"
-    "${CMAKE_SOURCE_DIR}/src/libunwind.zig"
-    "${CMAKE_SOURCE_DIR}/src/link.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/C.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/Coff.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/Coff/Atom.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/Coff/Object.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/Coff/lld.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/Elf.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/Archive.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/Atom.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/CodeSignature.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/DebugSymbols.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/DwarfInfo.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/Dylib.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/Object.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/Relocation.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/Trie.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/ZldAtom.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/bind.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/dead_strip.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/fat.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/load_commands.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/thunks.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/MachO/zld.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/Plan9.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/Plan9/aout.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/Wasm.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/msdos-stub.bin"
-    "${CMAKE_SOURCE_DIR}/src/link/strtab.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/tapi.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/tapi/Tokenizer.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/tapi/parse.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/tapi/parse/test.zig"
-    "${CMAKE_SOURCE_DIR}/src/link/tapi/yaml.zig"
-    "${CMAKE_SOURCE_DIR}/src/main.zig"
-    "${CMAKE_SOURCE_DIR}/src/mingw.zig"
-    "${CMAKE_SOURCE_DIR}/src/musl.zig"
-    "${CMAKE_SOURCE_DIR}/src/print_air.zig"
-    "${CMAKE_SOURCE_DIR}/src/print_env.zig"
-    "${CMAKE_SOURCE_DIR}/src/print_targets.zig"
-    "${CMAKE_SOURCE_DIR}/src/print_zir.zig"
-    "${CMAKE_SOURCE_DIR}/src/register_manager.zig"
-    "${CMAKE_SOURCE_DIR}/src/target.zig"
-    "${CMAKE_SOURCE_DIR}/src/tracy.zig"
-    "${CMAKE_SOURCE_DIR}/src/translate_c.zig"
-    "${CMAKE_SOURCE_DIR}/src/translate_c/ast.zig"
-    "${CMAKE_SOURCE_DIR}/src/type.zig"
-    "${CMAKE_SOURCE_DIR}/src/value.zig"
-    "${CMAKE_SOURCE_DIR}/src/wasi_libc.zig"
-    "${CMAKE_SOURCE_DIR}/src/windows_sdk.zig"
-)
+file(GLOB_RECURSE ZIG_STAGE2_SOURCES "${CMAKE_SOURCE_DIR}/lib/compiler_rt/*.zig")
+list(PREPEND ZIG_STAGE2_SOURCES "${ZIG_CONFIG_ZIG_OUT}")
```

Comment:
I intentionally do not use globbing here because new files and deleted files interact with CMake poorly.